src/preset_cli/cli/superset/sync/dbt/datasets.py

```diff
@@ -68,7 +80,7 @@ def create_dataset(
     return client.create_dataset(**kwargs)
 
 
-def sync_datasets(  # pylint: disable=too-many-locals, too-many-branches, too-many-arguments, too-many-statements
+def sync_datasets(  # pylint: disable=too-many-locals, too-many-branches, too-many-arguments, too-many-statements # noqa:C901
```

Comment:
This method is currently pretty large and has a lot of branches. Unsure if we wanna split it.

Reply:
Yeah, you're right.

Reply:
Alright, I can tackle this in a future PR!